stdlib/builtins.pyi

```diff
@@ -847,7 +847,7 @@ class dict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     def items(self) -> _dict_items[_KT, _VT]: ...
     @classmethod
     @overload
-    def fromkeys(cls, __iterable: Iterable[_T], __value: None = ...) -> dict[_T, Any | None]: ...
+    def fromkeys(cls, __iterable: Iterable[_T], __value: None = ...) -> dict[_T, object | None]: ...
```

Comment:
This also should remain `Any` for the same reason we initialize an empty dict as `dict[.., Any]`: Otherwise the new dict in unusable without casting. See the aioredis failures.

Reply:
And also some of the pytest stuff. Will revert.